authors/apps/articles/serializers.py

```diff
@@ -183,7 +179,32 @@ class CommentHistorySerializer(serializers.ModelSerializer):
     """
     This class handles the history of the comment edited
     """
-
     class Meta:
         model = Comments
         fields = ('id', 'body', 'created_at', 'updated_at')
+
```

Comment:
please add another line before  line 186 to adhere to PEP8 convention of two lines before a class.